IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -17,11 +17,19 @@
     </div>
 </a>
 
+
+@if (brew == null)
+{
+        <p> Breweries not found.</p>
+}
+
+else
+{
 <div class="text-center">
     <h1 class="display-4">Brewery List</h1>
     <hr />
     <div>

```

Comment:
Added a style class, instead of directly entering styles in the index page for best practices.

Reply:
Good.